.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: ruby
+rvm:
+  - 1.8.7
```

Comment:
But it still works! :) I am motivated to add all Rubies that currently work, even if it's not a goal to continue supporting them, for two reasons:
1) Customers can feel confident using this gem on whatever interpreter they have;
2) We make an informed decision about whether to accept a pull request / make a change knowing that it breaks an interpreter.